tests/scripts/all.sh

```diff
@@ -2351,7 +2351,7 @@ component_test_psa_crypto_config_accel_ecdh () {
 }
 
 component_test_psa_crypto_config_accel_ffdh () {
-    msg "build: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated FFDH"
+    msg "build: full with accelerated FFDH"
```

Comment:
I see other msg calls in all.sh that notify of the use of MBEDTLS_PSA_CRYPTO_CONFIG

Reply:
The short message should highlight what makes the build unique. “Accelerated FFDH” is the important thing here, and the fact that it's based on full is somewhat useful (it could plausibly be default instead). The fact that `MBEDTLS_PSA_CRYPTO_CONFIG` is involved is unimportant, it's just a tool to be able to enable acceleration.